students/ColeP/Session05/mailroom_3.py

```diff
@@ -0,0 +1,104 @@
+# Mailroom Yo
+
+donors = {"Donny Donor": [100, 10, 45],
+          "Gav Giver": [12, 16, 20],
+          "Stingy Steve": [2, 5, 1],
+          "Freaky Frank": [1200, 999, 1005],
+          }
+
+
+def thanks():
+    name_in = input('Who would you like to thank?\nPlease pick a name. Enter "List" to see names\n')
+    if name_in == 'List':
+        for i in donors:
+            print(i)
+        thanks()
+    elif name_in in donors:
```

Comment:
It may be useful to pull out the try/except and asking of the donation amount input before your if statement since this needs to happen in both.